src/Features/Core/Portable/Snippets/SnippetProviders/ISnippetProvider.cs

```diff
@@ -12,12 +12,17 @@ internal interface ISnippetProvider
         /// <summary>
         /// What we use to identify each SnippetProvider on the completion list
         /// </summary>
-        string SnippetIdentifier { get; }
+        string Shortcut { get; }
 
         /// <summary>
-        /// What is being displayed for the description of the snippet
+        /// What is being displayed for the inline-description of the snippet as well as the title on the tool-tip.
         /// </summary>
-        string SnippetDescription { get; }
+        string Title { get; }
+
+        /// <summary>
+        /// What is being displayed for the description of the snippet on the tool-tip.
+        /// </summary>
+        string Description { get; }
```

Comment:
If this is always in the same format of "Code snippet for [Title]", then it can be constructed by the snippet completion provider.